gaps/gap-1_gap_process/gap-1_gap_process.md

```diff
@@ -0,0 +1,44 @@
+---
+gap: 1
+title: GAP Process
+description: Specification of Golem Amendment Proposal process.
+author: stranger80 (@stranger80)>
+status: Draft
+type: Meta
+---
+
+## Abstract
+We are introducing a Golem Amendment Proposal process to establish governance over changes proposed, considered and implemented in Golem ecosystem.
+
+## Motivation
+The GAP process responds to a number of needs:
+- Establish governance over features, specification/standards amendments and process improvements managed by Golem Factory.
+- Ensure visibility of features in the Golem pipeline - this is especially valuable for interested parties outside of Golem Factory.
+- Provide ability for Golem community to submit proposals for features in a controlled, systematic manner.
+
+## Specification
+
+### GAP content & format
+
+- A GAP shall be raised as a pull request in `golem-architecture` repo titled along the lines of `GAP-draft Title`, including all the relevant artifacts (eg. GAP summary). 
+- A GAP summary `gap-draft_title.md` shall be provided (following a [template](../gap-template.md)) in `./gaps/gap-draft_title/`.
```

Comment:
```suggestion
- A GAP summary `gap-draft_title.md` shall be provided (following a [template](../gap-template.md)) in `./gaps/`.
```
I've looked in EIPs directory and there are flat files; I mean no subdir for each EIP.
They are using `assets` subdir when EIP needs more files. This approach reduces path redundancy proposed here, which iccurres repeatin `draft_title` in both filename and dirname, and also reduces friction for accessing GAP (one click: open file instead of two: open dir & open file).

Reply:
Actually I specifcally wanted all artifacts related to the GAP tobe in one folder - I don't like the idea of sending additional content to some other folder, I want everything in one place, that's what folders are for.